src/Infrastructure/DependencyInjection.cs

```diff
@@ -1,13 +1,13 @@
 ﻿using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
-using Application.Common.Interfaces;
-using Infrastructure.Database.Repositories;
-using Infrastructure.Database.Services;
+using CleanCompanyName.DDDMicroservice.Application.Common.Interfaces;
+using CleanCompanyName.DDDMicroservice.Infrastructure.Database.Repositories;
+using CleanCompanyName.DDDMicroservice.Infrastructure.Database.Services;
 using Mapster;
```

Comment:
`using`s from the same namespace (our namespace) are split here :)

Reply:
@jsellaresi, this is an example of that I was telling you about writing full namespaces even inside `namespace` block (even if IDE suggests it can be shorter) - sorting gets worse.

Reply:
Done